### PR TITLE
Create base repo folder for AnimeX (Abhi)

### PR DESCRIPTION
**Pull request by nasreekar**

### CODE REVIEW

**Comment by anandwana001** [comment]
@nasreekar Can we complete the Day 1 Task for building UI using Jetpack Compose in this PR?

**Comment by anandwana001** [comment]
Also, please remove the parent gitignore file

**Comment by nasreekar** [comment]
@anandwana001  removed the parent .gitignore file and added a dummy list view (not following the designs yet) but just dummy data

File: Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/components/AnimeCard.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    Surface(
+        color = MaterialTheme.colorScheme.primary,
+        modifier = modifier.padding(vertical = 4.dp, horizontal = 8.dp)
+    ) {
+        Column(
+            modifier = Modifier.fillMaxSize(1F),
+            verticalArrangement = Arrangement.SpaceAround,
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            SubcomposeAsyncImage(
```
Comment (anandwana001) [inline Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/components/AnimeCard.kt]: SubcomposeAsyncImage vs AsyncImage?

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/components/AnimeCard.kt (hunk 1 above)
Comment (nasreekar) [reply Abhi-AnimeX/app/src/main/java/com/abhijith/animex/ui/components/AnimeCard.kt]: Looks like my understanding of SubcomposeAsyncImage is wrong. i actually wanted it to show a loading indicator based on the state which subcompose provides but from the documentation it says that it is less performant especially in cases of lists. 

changing it to AsyncImage

**Comment by nasreekar** [comment]
# Pull Request

## Description

Day 2 - Update the Anime List Item UI to match the designs 
cont...
- Create Anime details page and add reusable components

## Checklist

- [x] I have added a screenshot or screen recording showing the functionality of the app.
- [x] I have tested the project and confirmed it works as expected.

## Screenshots

**Comment by anandwana001** [comment]
Can you take a pull from Master branch and update the PR description?

Also, do add the screenshot in visible height and width format

**Comment by nasreekar** [comment]
@anandwana001 apparently it is uptodate with upstream ... i tried fetching it and emerging but showing no changes

**Pull request merged**